specification/attributes/currency_code_format.md

```diff
@@ -0,0 +1,29 @@
+# Currency Code Format
+
+Columns that contain currency information in cost data following a consistent format reduces friction for FinOps practitioners that consume the data for analysis, reporting, and other use cases.
+
+All currency related columns defined in the FOCUS specification MUST follow the formatting requirements listed below. Provider generated currency-related columns SHOULD adopt the same format requirements over time.
```

Comment:
nit: "FOCUS specification" is redundant. Should we use "schema" or just leave it out?
```suggestion
All currency related columns defined in the FOCUS schema MUST follow the formatting requirements listed below. Provider generated currency-related columns SHOULD adopt the same format requirements over time.
```